.github/workflows/pull_requests.yml

```diff
@@ -0,0 +1,32 @@
+name: PR Action
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  jdk17:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: maven
+
+      - name: JDK 17 Tests
+        run: mvn -B verify -Dmaven.test.failure.ignore=true -Dansi.strip=true
+
+      - name: Test Logs
```

Comment:
In order for these results to be displayed you also need the test report action:

https://github.com/infinispan/protostream/blob/main/.github/workflows/test_report.yaml

Reply:
Do you see something wrong? The report action is not triggering. Maybe only triggers when the event happens in main?

Reply:
Nothing wrong, `test_report.yaml` needs to be merged for it to trigger.